backend/prisma/schema.prisma

```diff
@@ -0,0 +1,37 @@
+// This is your Prisma schema file,
+// learn more about it in the docs: https://pris.ly/d/prisma-schema
+
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "sqlite"
+  url      = env("DATABASE_URL")
+}
+
+model User {
+  id        Int     @id @default(autoincrement())
```

Comment:
typically we'd want to use UUID instead of autoincrement for the user table. UUID is globally unique meaning no two id will ever be the same, and there are many other security reasons behind it too. so use uuid here.